Common subdirectories: a/MonoGame.Framework/Graphics/Effect and b/MonoGame.Framework/Graphics/Effect
Common subdirectories: a/MonoGame.Framework/Graphics/PackedVector and b/MonoGame.Framework/Graphics/PackedVector
diff -u a/MonoGame.Framework/Graphics/SpriteEffects.cs b/MonoGame.Framework/Graphics/SpriteEffects.cs
--- a/MonoGame.Framework/Graphics/SpriteEffects.cs	2012-03-27 10:37:14.000000000 -0700
+++ b/MonoGame.Framework/Graphics/SpriteEffects.cs	2012-04-29 18:21:08.159997186 -0700
@@ -40,7 +40,7 @@
 
 using System;
 
-﻿namespace Microsoft.Xna.Framework.Graphics
+namespace Microsoft.Xna.Framework.Graphics
 {
     [Flags]
     public enum SpriteEffects
Common subdirectories: a/MonoGame.Framework/Graphics/States and b/MonoGame.Framework/Graphics/States
diff -u a/MonoGame.Framework/Graphics/SwapEffect.cs b/MonoGame.Framework/Graphics/SwapEffect.cs
--- a/MonoGame.Framework/Graphics/SwapEffect.cs	2012-03-27 10:37:14.000000000 -0700
+++ b/MonoGame.Framework/Graphics/SwapEffect.cs	2012-04-29 18:21:26.419998039 -0700
@@ -40,7 +40,7 @@
 
 using System;
 
-﻿namespace Microsoft.Xna.Framework.Graphics
+namespace Microsoft.Xna.Framework.Graphics
 {
     public enum SwapEffect
     {
Common subdirectories: a/MonoGame.Framework/Graphics/Vertices and b/MonoGame.Framework/Graphics/Vertices
